### PR TITLE
Accounts: Remove references to the old v1 user model

### PR DESCRIPTION
**Pull request by rhetoric101**

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx [hunk 1]
```diff
@@ -61,10 +58,9 @@ If you're logged in but can't find an expected account or UI page, it may be bec
 * If you have multiple user records (for example, access to more than one New Relic organization), you can [switch between your user records](/docs/accounts/accounts-billing/account-setup/multiple-logins-found).
 * If you can't find an expected account, it may be because you haven't been assigned access to that account. See [user permissions issues](#user-permissions).
 * If you check the account switcher and can't see the account you're looking for, it may be for one of these reasons:
-  * You may need to be added to that account. How you do this depends on your user model: [Our newer user model](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#account-access) \| [Our original user model](/docs/accounts/original-accounts-billing/original-users-roles/original-account-structure).
+  * You may need to be added to that account. See [How users access accounts](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/#account-access).
```
Comment (jeff-colucci) [inline src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx:61]: Passive voice here, maybe something like "You may need an administrator to add you to the account." or something along those lines? Also if you wind up making an edit to this line, the second sentence ends a bit abruptly, maybe add a "for more information" or something generic at the end?

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions.mdx [hunk 2]
```diff
@@ -6,25 +6,29 @@ tags:
...
 
-There are a lot of New Relic functionalities that we <DNT>**don't**</DNT> make visible and available for selection. For example, there are various UI pages that you can access as any user, and that aren't gated by the permissions we expose. For another example: a user in a group with <DNT>**Organization**</DNT> [administration settings](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings) can configure organization-level settings and that power is not available for adding to a custom role.
+There are a lot of New Relic functionalities that we don't make visible and available as permissions you could select. For example, there are various UI pages that you can access as any user and that aren't gated by the permissions we expose. For another example: A user in a group with <DNT>**Organization**</DNT> [administration settings](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings) can configure organization-level settings and that power is not available for adding to a custom role.
```
Comment (jeff-colucci) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions.mdx:26]: As vale says, is not > isn't

**Review by jeff-colucci — DISMISSED** [review]
I only found some small changes to be made, happy to approve this now or re-approve after you make them!

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions.mdx:26]: @jeff-colucci sorry I missed your review. Could you explain this suggestion?

File: src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data.mdx:61]: @jeff-colucci I've pushed up two revisions. Thanks!

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-saml-scim.mdx [hunk 3]
```diff
@@ -11,25 +12,7 @@ For setting up automatic controls over how your New Relic users are added to New
...
 * <DNT>**SCIM provisioning**</DNT>, also referred to as automated user management: SCIM provisioning allows organizations to use their identity provider service to automate how their users are added to and updated in New Relic. Organizations using SCIM will almost always use SAML SSO. Requires [Pro or Enterprise edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions).
 
-For information on how to enable SAML and SCIM, choose your user model below. If you're not sure which you're on, see [Overview of user models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models).
+For information on how to enable SAML and SCIM, you'll use the [authentication domain UI](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more) to set up SAML and SCIM. Note that Okta, OneLogin, and Azure have dedicated New Relic apps: If you use those, you must choose the one called **New Relic by organization**, and not **New Relic by account**.
```
Comment (jeff-colucci) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-saml-scim.mdx:15]: I'm not sure if the If in the latter part of the sentence should be capitalized?

Comment (rhetoric101) [reply src/content/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-saml-scim.mdx:15]: @jeff-colucci I think we're only capitalizing following a colon if there's a complete sentence.

**Review by jeff-colucci — APPROVED** [review]

**Review by paperclypse — APPROVED** [review]
All preview URLs loaded successfully. Looks good! A mammoth effort

**Comment by rhetoric101** [comment]
I had to override the approval since GitHub was not in synch with Netlify. This kicked off an automated Jira: https://new-relic.atlassian.net/browse/NR-339219

I am closing this Jira.

**Pull request merged**